### PR TITLE
release 0.10.8

### PR DESCRIPTION
**Pull request by Javakky-pxv**
@kailuowang 
I have modified the version string and readme to match the next version and would like to see the changes so far published!

### CODE REVIEW

**Comment by Javakky-pxv** [comment]
Pull Requests that were piling up have been resolved in their entirety.

File: version.sbt [hunk 1]
```diff
@@ -1 +1 @@
-ThisBuild / version := "0.10.8-SNAPSHOT"
+ThisBuild / version := "0.10.8"
```
Comment (kailuowang) [inline version.sbt:1]: you don't need to change this settings. The release process updates it automatically.

File: example/project/plugins.sbt [hunk 2]
```diff
@@ -6,4 +6,4 @@ addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.6")
 addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.10.1")
 
 // play swagger plugin
-addSbtPlugin("com.iheart" % "sbt-play-swagger" % "0.10.8-SNAPSHOT")
+addSbtPlugin("com.iheart" % "sbt-play-swagger" % "0.10.8")
```
Comment (kailuowang) [inline example/project/plugins.sbt:9]: this should be 0.10.9-SNAPSHOT

**Comment by kailuowang** [comment]
I need to do the release before merging this. Just want to confirm with you that you want a release of the current master branch right?

**Comment by Javakky-pxv** [comment]
@kailuowang 
I would like to see the current latest master released as 0.10.8.
Are there any changes that need to be made to achieve this?
I expected this PR to be it.

**Comment by kailuowang** [comment]
Just released 0.10.8
feel free the merge this once you fix the comment above. Thanks!

**Pull request merged**